### PR TITLE
some test_context fixes

### PR DESCRIPTION
**Pull request by tyrantlizard77**
Low-hanging fruit when examining ZEP-2243.

### CODE REVIEW

File: tests/kernel/context/src/context.c [hunk 1]
```diff
@@ -189,7 +189,7 @@ static int kernel_init_objects(void)
  */
 static int test_kernel_cpu_idle(int atomic)
 {
-	int tms;                /* current time in millisecond */
+	int tms, tms2;;         /* current time in millisecond */
```
Comment (lpereira) [inline tests/kernel/context/src/context.c]: Stray semicolon.

File: tests/kernel/context/src/context.c [hunk 2]
```diff
@@ -351,24 +361,39 @@ static int test_kernel_ctx_task(void)
...
+	}
+
+	if (isr_info.value != K_ISR) {
+		TC_ERROR("isr_info.value was not K_ISR\n");
 		return TC_FAIL;
 	}
 
-	TC_PRINT("Testing k_is_in_isr() from a preemtible thread\n");
-	if (k_is_in_isr() || _current->base.prio < 0) {
+	TC_PRINT("Testing k_is_in_isr() from a preemptible thread\n");
```
Comment (lpereira) [inline tests/kernel/context/src/context.c]: Also update README.txt (it contains the output with the typo).

**Review by lpereira — APPROVED** [review]
LGTM with nits

**Comment by nashif** [comment]
This does not fix the issue raised in ZEP-2243 on Arduino Due

**Comment by andrewboie** [comment]
@nashif no it doesn't. But it fixes another issue reproducible intermittently in QEMU, even if the system is not under heavy load. I am still researching ZEP-2243.

**Pull request merged**